ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Ledger/Tables/Combinators.hs

```diff
@@ -52,6 +54,10 @@ module Ouroboros.Consensus.Ledger.Tables.Combinators (
   , fn2_3
   , fn2_4
   , type (-..->) (..)
+    -- ** Re-exports of utils
+  , (...:)
+  , (..:)
+  , (.:)
```

Comment:
I added this re-export because we often need this if we want to use `ltcollapse`. Example: https://github.com/input-output-hk/ouroboros-consensus/pull/183/files#diff-0ed0aed79381e4ab7095e17161c46717b17456e82dd5992cc7a9c9ae9219deadR341